src/redisearch_rs/inverted_index/src/tests.rs

```diff
@@ -2004,3 +2008,54 @@ fn ii_apply_gc_entries_tracking_index() {
         }
     );
 }
+
+#[test]
+fn test_refresh_buffer_pointers_after_reallocation() {
+    let mut ii = InvertedIndex::<Dummy>::new(IndexFlags_Index_DocIdsOnly);
```

Comment:
```suggestion
// the memory hack below raises error in miri
#[cfg(not(miri))] 
#[test]
fn test_refresh_buffer_pointers_after_reallocation() {
    let mut ii = InvertedIndex::<Dummy>::new(IndexFlags_Index_DocIdsOnly);
```

This should fix the CI.